yeps/yep-310.md

```diff
@@ -19,21 +20,48 @@ Daemons implementing this trait allow measurement to be software triggered over
 # Motivation
 
 In many cases, it's relevant to tell a sensor "begin measurement now".
-This software triggered measurement pattern is very common.
+For example, you might wish to wait for a motor to arrive at its position before telling a sensor to make a measurement.
 This trait standardizes this common sensor interface pattern within the yaq ecosystem.
 
+Note that this trait builds on the is-sensor trait defined in [YEP-302](https://yeps.yaq.fyi/302/).
+That trait defines a standard way of accessing the sensor data and metadata.
+The scope of this trait is only the software triggering feature.
+
 # Proposal
 
 This trait requires is-sensor: see [YEP-302](https://yeps.yaq.fyi/302/).
 
-This trait defines the concept of "looping".
-When a sensor is looping, it should self-trigger the moment the previous acquisition is done.
-In certain cases, setting a sensor to looping makes client implementations much easier.
-Daemons will remain always busy while looping.
-
-This trait defines the concept of a "measurement_id".
-This is an integer that should increase every time the measure message is called.
-`get_measured` returns this measurement_id as a channel.
+The principle feature of the has-measure-trigger is the addition of the `measure` method.
+This method can be called to initiate a measurement.
+Because measurements might take some time to complete, the `measure` method does not wait to return the results of the measurement.
+Instead, it merely returns the `measurement_id` of the measurement that was initiated (the "next" measurement).
+Clients will use the messages defined by [YEP-302](https://yeps.yaq.fyi/302/) to access the data once the measurement is complete.
+Clients will be assured of measurement order by comparing the `measurement_id` returned by `get_measured` to tme `measurement_id` that was returned by `measure` previously.
```

Comment:
The original concept (the one I liked) was for `get_measured` to return the new measurement_id from `measure` once all data was captured.    Otherwise, how would one know when data capture is completed?

Reply:
I agree! That's the behavior I tried to describe here, but clearly I failed. I'll make sure to change this language so it's clearer!

Reply:
I think just a line between 38 and 39 saying that once the new measurement completes, the new id from the `measure` will be pasted to the new data.